lib/util.js

```diff
@@ -740,6 +740,10 @@ exports.getEnvVar = function (variable) {
   return process.env[variable.toLowerCase()] || process.env[variable.toUpperCase()];
 };
 
+exports.restoreEnvVar = function (envVar, value) {
```

Comment:
This function is for the Azure proxy and I made testing cases in the Azure PR. Because of the 'useEnvProxy' option, we still need this function. Here are the testing case: https://github.com/snowflakedb/snowflake-connector-nodejs/blob/08d802e696a7c3a594dc0e4935279e014d9ff5f1/test/unit/proxy_util_test.js#L301